source/agora/test/Base.d

```diff
@@ -172,6 +172,49 @@ private UnitTestResult customModuleUnitTester ()
     return result;
 }
 
+/// A custom serializer for LocalRest
+public struct Serializer
+{
+    import agora.common.Serializer;
+    import std.stdio;
+
+    static immutable(ubyte)[] serialize (T) (auto ref T value) @trusted nothrow
+    {
+        // `serializeFull` should be `@safe`, but `assumeUnique` is not
+        try
+            return ((arr) @trusted => assumeUnique(arr))(serializeFull(value));
+        catch (Throwable t)
+        {
+            try
+            {
+                writeln("ERROR: Serializable: ", T.stringof);
+                writeln(t);
+            }
+            catch (Exception ignored) {}
+            assert(0);
+        }
+    }
+
+    static QT deserialize (QT) (scope immutable(ubyte)[] data) @trusted nothrow
+    {
+        try
+            return deserializeFull!QT(data);
+        catch (Throwable t)
+        {
+            try
+            {
+                writeln("ERROR: Deserialize:", QT.stringof);
+                writeln(t);
+            }
+            catch (Exception ignored) {}
+            assert(0);
+        }
+    }
+}
+
+/// A different default serializer from `LocalRest` for `RemoteAPI`
+alias RemoteAPI (APIType) = geod24.LocalRest.RemoteAPI!(APIType, Serializer);
```

Comment:
I think the code in `agora.test.NetworkManager` will still use the default one that uses Json serialization, because it uses a local import into LocalRest.. which will probably hide this symbol here.

Reply:
The PR also removes that import

Reply:
Oh didn't see. What about also making the import in this module static? I don't think any other symbols are used from it except here?